src/browser/modules/Stream/SchemaFrame.tsx

```diff
@@ -163,13 +192,17 @@ export class SchemaFrame extends Component<any, SchemaFrameState> {
     }
   }
 
-  fetchData() {
+  fetchData(neo4jVersion: string) {
     if (this.props.bus) {
       // Indexes
       this.props.bus.self(
         CYPHER_REQUEST,
         {
-          query: 'CALL db.indexes()',
+          query:
+            semver.valid(neo4jVersion) &&
```

Comment:
There is a a `getSemanticVersion` getter in `dbMetaDuck` you can use instead of `getRawVersion`. You will still have to do some checks, but it calls `semver.coerce` which I think is could be needed for some odd version numbers that aura uses

Reply:
I'll check that it works properly against my aura instance :+1: